docs/resources/new_cluster.plantuml

```diff
@@ -0,0 +1,24 @@
+@startuml
+title Add new cluster
+
+actor "OSIO user" as user
+box "OSD Cluster" #LightBlue
+	participant "cluster API" as cluster
+end box
+box "OSIO"
+	participant "Cluster Service" as cs
+	participant "Auth Service" as auth
+	participant "UI\nuser settings page" as usersettings
+end box
```

Comment:
I got little confused to understand when req/res (req from left to right and res from right to left) calls are running in other direction than usual.  I see that if we flip position of `OSD Cluster` and `OSIO` boxes then overall seq_flow will become easy to navigate and understand.

Reply:
Good point. I actually created one diagram with all three flows then split them in three without reordering the boxes which made it look weird.